presentation/src/main/java/me/androidbox/presentation/photo/screen/PhotoScreenState.kt

```diff
@@ -0,0 +1,5 @@
+package me.androidbox.presentation.photo.screen
+
+data class PhotoScreenState(
+    val photoSelected: String = ""
```

Comment:
Naming recommendation: `selectedPhoto` and make it of type `EventPhoto` (or whatever you use for unifying local and remote photos)